.github/workflows/scripts/common/escapes.py

```diff
@@ -0,0 +1,58 @@
+"""
+Generates a dictionary of ANSI escape codes.
+
+http://en.wikipedia.org/wiki/ANSI_escape_code
+
+Uses colorama as an optional dependency to support color on Windows
+
+---
+Copied from colorlog
+---
```

Comment:
Hmm is it only used for pretty banner (and not related to ci/test)? If so let's add a simple line here explaining it's for banner only ;)